app/controllers/cases/offender_sar_controller.rb

```diff
@@ -29,6 +29,7 @@ def initialize
 
     def new
       permitted_correspondence_types
+      @is_rejected = !!params.fetch("is_rejected", false) # rubocop:disable Style/DoubleNegation
```

Comment:
To avoid the rubocop complaint, it might be better to use `key?` instead:
```suggestion
      @is_rejected = params.key?("is_rejected")
```

Reply:
A very minor one, but as it's user facing, maybe the URL param should be `rejected` instead of `is_rejected`?